[SIN-25] Announce 2025 edition

sergiubodiu opened the pull request:
[SIN-2025] Announce Dates and location
[SIN-2024] Add presentation decks and recorded videos.

sergiubodiu commented:
Let me know if you have any questions, we're announcing 2025 edition for 14-15 May, have a good weekend.

sergiubodiu commented:
Added the logo square using new format.

toshywoshy commented on static/events/2025-singapore/logo.png:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Noted, I didn’t know about the new convention, moved this and others from static to assets.

toshywoshy commented on static/events/2025-singapore/organizers/diana-bodiu.jpg:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Noted, moved.

toshywoshy commented on static/events/2025-singapore/organizers/lin-ling.jpg:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Noted, moved

toshywoshy commented on static/events/2025-singapore/organizers/sergiu-bodiu.jpg:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Noted, moved

toshywoshy commented on static/events/2025-singapore/organizers/tamara-draguta.jpg:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Noted, moved.

toshywoshy commented on static/events/2025-singapore/skyline-merlion.jpg:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Noted, moved

toshywoshy commented on static/img/sponsors/ttabsg.png:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Noted, moved to assets/sponsors/t/ttabsg.png

toshywoshy commented on static/events/2025-singapore/logo-square.jpg:
You already have this file in the assets, there is no need for this file anymore

sergiubodiu replied:
Removed, we were using the location in static, and saw that the event logo is not showing in events page.

toshywoshy commented on static/events/2024-singapore/organizers/tamara-draguta.jpg:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Moved

toshywoshy commented on static/events/2024-singapore/organizers/lin-ling.jpg:
Please move to the new location `assets/events/2025-singapore/`

sergiubodiu replied:
Moved

toshywoshy commented on data/events/2024/singapore/main.yml:
These are changes to the previous year, is this what you want?

sergiubodiu replied:
Yes, we updates the profile pictures, and we also updated the order of the ignites as they were recorded

toshywoshy commented on content/events/2024-singapore/welcome.md:
Why are you changing the old page, you're adding 2025, there is no need to change the old page

sergiubodiu replied:
After the 2024 conclusion, changed the wording and inviting people to look at the playlist and speaker decks.

toshywoshy commented:
A note for future reference, it would better to have the 2024 updates/changes in a separate PR.
This would mean that that PR is not dependant on other changes.

sergiubodiu commented:
Moved static images to new assets folder. For clarification we closed 2024, added the video recordings and speaker decks, changed the wording showing that it was wrapped  and are announcing the 2025 edition

jyee commented on content/events/2025-singapore/location.md:
```diff
@@ -0,0 +1,96 @@
+
+Find more facts about Singapore here:
+
+    https://en.wikipedia.org/wiki/Singapore
```
The indentation here is probably not what you were intending, since that turns the text into a code block and they're not hyperlinked.

jyee reviewed (approved):
Looks like everything from @toshywoshy's review was corrected.
I left a couple minor notes. I don't think either are blocking, but worth flagging for @sergiubodiu.
